### PR TITLE
Add Brand ID To Provider Config

### PR DESCRIPTION
**Pull request by RoyalSix**
## Description

[Pull Request Description Here]

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/react-inbox/src/constants.ts [hunk 1]
```diff
@@ -2,3 +2,18 @@ export const MESSAGE_LABELS = {
   MARK_AS_READ: "Mark as Read",
   MARK_AS_UNREAD: "Mark as Unread",
 };
+
+export const DEFAULT_TABS = [
```
Comment (rileylnapier) [inline packages/react-inbox/src/constants.ts:6]: 👍

File: packages/react-provider/src/hooks/use-brand.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    const [results] = useQuery({
+      query: GET_BRAND,
+      variables: {
+        brandId,
+      },
+    });
+    debugger;
+    setBrand(results?.data);
+  }
+  console.log("brand", brand);
```
Comment (rileylnapier) [inline packages/react-provider/src/hooks/use-brand.ts:28]: remove?

File: packages/storybook/stories/react-inbox/with-toast.stories.tsx [hunk 3]
```diff
@@ -52,6 +52,7 @@ export function Default() {
         clientKey={clientKey}
         userId={userId}
         transport={courierTransport}
+        brandId="S0E8PFMXC44SSFG6J8A7EF4QXRNX"
```
Comment (rileylnapier) [inline packages/storybook/stories/react-inbox/with-toast.stories.tsx:55]: might want an env variable so we can change between our dev envs

Comment (RoyalSix) [reply packages/storybook/stories/react-inbox/with-toast.stories.tsx]: Done

File: packages/react-provider/src/hooks/use-brand.ts (hunk 2 above)
Comment (RoyalSix) [reply packages/react-provider/src/hooks/use-brand.ts]: Done

File: packages/react-provider/src/index.tsx [hunk 4]
```diff
@@ -52,29 +58,37 @@ export const CourierProvider: React.FunctionComponent<ICourierContext> = ({
...
+          apiUrl,
+          brandId,
+          clientKey,
+          transport,
+          userId,
+          userSignature,
+        },
+      });
+      if (brandId) {
+        dispatch({
```
Comment (rileylnapier) [inline packages/react-provider/src/index.tsx:85]: this should be in its own useEffect

File: packages/react-provider/src/reducer.ts [hunk 5]
```diff
@@ -33,6 +33,13 @@ const rootReducer = (state, action) => {
         transport: action.payload.transport,
         userId: action.payload.userId,
         userSignature: action.payload.userSignature,
+        brandId: action.payload.brandId,
+      };
+    }
+    case "root/GET_BRAND/DONE": {
```
Comment (rileylnapier) [inline packages/react-provider/src/reducer.ts:39]: what do we do while we are fetching a brand?  we prob don't want to render anything right?

File: packages/react-provider/src/index.tsx [hunk 6]
```diff
@@ -88,15 +102,14 @@ export const CourierProvider: React.FunctionComponent<ICourierContext> = ({
...
-
   return (
     <CourierContext.Provider
       value={{
         ...(state as any),
         dispatch,
       }}
     >
-      <GraphQLProvider>{children}</GraphQLProvider>
+      <GraphQLProvider client={client}>{children}</GraphQLProvider>
```
Comment (rileylnapier) [inline packages/react-provider/src/index.tsx:112]: why do we need to pass client in here?

**Review by rileylnapier — APPROVED** [review]

**Pull request merged**